src/utils/AutoControlledComponent-slim.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-console */
```

Comment:
Review this file, this is identical to actual component, but with no comments nor run time warnings.  It is only the production code.  This makes it about 1/3 the size and much easier to grok.